# Change Log

## [Unreleased](https://github.com/sanger-pathogens/deployment/tree/HEAD)

**Merged pull requests:**

- add crontabs to deployment [\#200](https://github.com/sanger-pathogens/deployment/pull/200) ([puethe](https://github.com/puethe))
-  add wrapper script for back-up of Chado audit tables  [\#199](https://github.com/sanger-pathogens/deployment/pull/199) ([puethe](https://github.com/puethe))
- removed script genedb\_dump\_database\_upload\_to\_ftp [\#198](https://github.com/sanger-pathogens/deployment/pull/198) ([ssjunnebo](https://github.com/ssjunnebo))
- update test mapping path [\#197](https://github.com/sanger-pathogens/deployment/pull/197) ([vaofford](https://github.com/vaofford))
- add pathdev\_config and database connection details [\#196](https://github.com/sanger-pathogens/deployment/pull/196) ([vaofford](https://github.com/vaofford))
- added python script 'eupathdb\_stats.py' [\#195](https://github.com/sanger-pathogens/deployment/pull/195) ([puethe](https://github.com/puethe))
- two wrapper scripts for chado-related cron tasks; URL of git repo for… [\#194](https://github.com/sanger-pathogens/deployment/pull/194) ([puethe](https://github.com/puethe))
- removed old 'chado' script [\#193](https://github.com/sanger-pathogens/deployment/pull/193) ([puethe](https://github.com/puethe))
- 574328 Set up Travis [\#192](https://github.com/sanger-pathogens/deployment/pull/192) ([ssjunnebo](https://github.com/ssjunnebo))
- add AlignGraph to shell/scripts [\#191](https://github.com/sanger-pathogens/deployment/pull/191) ([vaofford](https://github.com/vaofford))
- add wrapper to shell/scripts for graphtyper RT\_626185 [\#190](https://github.com/sanger-pathogens/deployment/pull/190) ([vaofford](https://github.com/vaofford))
- add back fasta\_extractor.pl [\#189](https://github.com/sanger-pathogens/deployment/pull/189) ([ssjunnebo](https://github.com/ssjunnebo))
- remove further scripts and roll back psu\_union.pl [\#188](https://github.com/sanger-pathogens/deployment/pull/188) ([ssjunnebo](https://github.com/ssjunnebo))
- remove dotplot and sequence\_search [\#187](https://github.com/sanger-pathogens/deployment/pull/187) ([ssjunnebo](https://github.com/ssjunnebo))
- remove rfamscan\_dechunk.pl [\#186](https://github.com/sanger-pathogens/deployment/pull/186) ([ssjunnebo](https://github.com/ssjunnebo))
- remove obsolete scripts [\#185](https://github.com/sanger-pathogens/deployment/pull/185) ([ssjunnebo](https://github.com/ssjunnebo))
- add snippy wrapper for RT 618973 [\#184](https://github.com/sanger-pathogens/deployment/pull/184) ([vaofford](https://github.com/vaofford))
- wrapper for graphmap - RT618584 [\#183](https://github.com/sanger-pathogens/deployment/pull/183) ([vaofford](https://github.com/vaofford))
- breseq-0.32.0 wrapper RT618212 [\#182](https://github.com/sanger-pathogens/deployment/pull/182) ([vaofford](https://github.com/vaofford))
- Removed Artemis, ACT, DNA Plotter related scripts as part of v17.0.1 … [\#181](https://github.com/sanger-pathogens/deployment/pull/181) ([kpepper](https://github.com/kpepper))
- add racon wrapper to shell/scripts [\#180](https://github.com/sanger-pathogens/deployment/pull/180) ([vaofford](https://github.com/vaofford))
- add wrapper for fqtools to shell/scripts [\#179](https://github.com/sanger-pathogens/deployment/pull/179) ([vaofford](https://github.com/vaofford))
- add scripts/fastANI for RT 602806 [\#178](https://github.com/sanger-pathogens/deployment/pull/178) ([vaofford](https://github.com/vaofford))
- add wrapper for filtlong RT 595617 [\#177](https://github.com/sanger-pathogens/deployment/pull/177) ([vaofford](https://github.com/vaofford))
- utr\_analysis wrapper script - RT 593858 [\#176](https://github.com/sanger-pathogens/deployment/pull/176) ([vaofford](https://github.com/vaofford))
- add version to tophat wrapper [\#175](https://github.com/sanger-pathogens/deployment/pull/175) ([vaofford](https://github.com/vaofford))
- add tophat wrapper to link tophat 2.1.0 with bowtie 2.1.0 [\#174](https://github.com/sanger-pathogens/deployment/pull/174) ([vaofford](https://github.com/vaofford))
- ngsDist [\#173](https://github.com/sanger-pathogens/deployment/pull/173) ([andrewjpage](https://github.com/andrewjpage))
- add mashmap wrapper [\#172](https://github.com/sanger-pathogens/deployment/pull/172) ([vaofford](https://github.com/vaofford))
- add setup\_seroba\_dbs.sh [\#171](https://github.com/sanger-pathogens/deployment/pull/171) ([vaofford](https://github.com/vaofford))
- brig wrapper RT582624 [\#170](https://github.com/sanger-pathogens/deployment/pull/170) ([vaofford](https://github.com/vaofford))
- added shell/scripts/download\_ariba\_databases.sh [\#169](https://github.com/sanger-pathogens/deployment/pull/169) ([vaofford](https://github.com/vaofford))
- add tracer.conf [\#168](https://github.com/sanger-pathogens/deployment/pull/168) ([vaofford](https://github.com/vaofford))
- Trinity wrapper to shell/scripts [\#167](https://github.com/sanger-pathogens/deployment/pull/167) ([vaofford](https://github.com/vaofford))
- Unicycler wrapper [\#166](https://github.com/sanger-pathogens/deployment/pull/166) ([vaofford](https://github.com/vaofford))
- add strainphlan wrapper [\#165](https://github.com/sanger-pathogens/deployment/pull/165) ([vaofford](https://github.com/vaofford))
- Add script fc\_run [\#164](https://github.com/sanger-pathogens/deployment/pull/164) ([martinghunt](https://github.com/martinghunt))
- Update global.yml [\#163](https://github.com/sanger-pathogens/deployment/pull/163) ([ssjunnebo](https://github.com/ssjunnebo))
- update artemis script location [\#162](https://github.com/sanger-pathogens/deployment/pull/162) ([vaofford](https://github.com/vaofford))
- Move canu [\#161](https://github.com/sanger-pathogens/deployment/pull/161) ([vaofford](https://github.com/vaofford))
- changed shell/scripts/run\_canu.sh to shell/scripts/canu.sh [\#160](https://github.com/sanger-pathogens/deployment/pull/160) ([vaofford](https://github.com/vaofford))
- added scripts/scripts/prepare\_artemis\_release.sh [\#159](https://github.com/sanger-pathogens/deployment/pull/159) ([vaofford](https://github.com/vaofford))
- Remove average\_nucleotide\_identity.py [\#158](https://github.com/sanger-pathogens/deployment/pull/158) ([martinghunt](https://github.com/martinghunt))
- Add perl/scripts/het\_snp\_calculator.pl [\#157](https://github.com/sanger-pathogens/deployment/pull/157) ([martinghunt](https://github.com/martinghunt))
- added shell/scripts/orderedPainting [\#156](https://github.com/sanger-pathogens/deployment/pull/156) ([vaofford](https://github.com/vaofford))
- gene db database to ftp script [\#155](https://github.com/sanger-pathogens/deployment/pull/155) ([andrewjpage](https://github.com/andrewjpage))
- added sourmash wrapper [\#154](https://github.com/sanger-pathogens/deployment/pull/154) ([vaofford](https://github.com/vaofford))
- add run\_blastp\_jobs.pl [\#153](https://github.com/sanger-pathogens/deployment/pull/153) ([vaofford](https://github.com/vaofford))
- atlas wrapper [\#152](https://github.com/sanger-pathogens/deployment/pull/152) ([andrewjpage](https://github.com/andrewjpage))
- ani wrapper [\#151](https://github.com/sanger-pathogens/deployment/pull/151) ([andrewjpage](https://github.com/andrewjpage))
- added shovill [\#150](https://github.com/sanger-pathogens/deployment/pull/150) ([psweston](https://github.com/psweston))
- Added pilon wrapper [\#149](https://github.com/sanger-pathogens/deployment/pull/149) ([psweston](https://github.com/psweston))
- Added new recycler wrapper recycle.py [\#148](https://github.com/sanger-pathogens/deployment/pull/148) ([psweston](https://github.com/psweston))
- add extra Python scripts requested by Uli [\#147](https://github.com/sanger-pathogens/deployment/pull/147) ([satta](https://github.com/satta))
- add Lua scripts [\#146](https://github.com/sanger-pathogens/deployment/pull/146) ([satta](https://github.com/satta))
- Added missing comma [\#145](https://github.com/sanger-pathogens/deployment/pull/145) ([psweston](https://github.com/psweston))
- Added new file scripts/mrbayes [\#144](https://github.com/sanger-pathogens/deployment/pull/144) ([psweston](https://github.com/psweston))
- lcmlkin wrapper [\#143](https://github.com/sanger-pathogens/deployment/pull/143) ([andrewjpage](https://github.com/andrewjpage))
- Add script runMetaBat.sh [\#142](https://github.com/sanger-pathogens/deployment/pull/142) ([martinghunt](https://github.com/martinghunt))
- rapmap [\#141](https://github.com/sanger-pathogens/deployment/pull/141) ([andrewjpage](https://github.com/andrewjpage))
- wrapper scripts for w2rap-contigger and beast [\#140](https://github.com/sanger-pathogens/deployment/pull/140) ([andrewjpage](https://github.com/andrewjpage))
- account monitor [\#139](https://github.com/sanger-pathogens/deployment/pull/139) ([andrewjpage](https://github.com/andrewjpage))
- Added filter\_for\_reago.sh script to prod bin [\#138](https://github.com/sanger-pathogens/deployment/pull/138) ([aslett1](https://github.com/aslett1))
- Add mapping for config for pf script [\#137](https://github.com/sanger-pathogens/deployment/pull/137) ([jgtate](https://github.com/jgtate))
- Added canu wrapper script [\#136](https://github.com/sanger-pathogens/deployment/pull/136) ([aslett1](https://github.com/aslett1))
- Create checksums of deployed artifacts [\#135](https://github.com/sanger-pathogens/deployment/pull/135) ([bewt85](https://github.com/bewt85))
- Added phylosift wrapper script [\#134](https://github.com/sanger-pathogens/deployment/pull/134) ([aslett1](https://github.com/aslett1))
- Add script to backup MIDAS audit logs [\#133](https://github.com/sanger-pathogens/deployment/pull/133) ([jgtate](https://github.com/jgtate))
- Added repeat\_finder.pl to production bin [\#132](https://github.com/sanger-pathogens/deployment/pull/132) ([aslett1](https://github.com/aslett1))
- new perl script [\#131](https://github.com/sanger-pathogens/deployment/pull/131) ([nds](https://github.com/nds))
- Added mash wrapper script to prod deployments [\#130](https://github.com/sanger-pathogens/deployment/pull/130) ([aslett1](https://github.com/aslett1))
- Removed legacy iprscan script from deployment [\#129](https://github.com/sanger-pathogens/deployment/pull/129) ([aslett1](https://github.com/aslett1))
- add Uniprot script [\#128](https://github.com/sanger-pathogens/deployment/pull/128) ([satta](https://github.com/satta))
- Deployment of ena\_data\_submission.conf [\#127](https://github.com/sanger-pathogens/deployment/pull/127) ([aslett1](https://github.com/aslett1))
- Removed validate\_embl.pl from perl prod deployments [\#126](https://github.com/sanger-pathogens/deployment/pull/126) ([aslett1](https://github.com/aslett1))
- fix typo in script name [\#125](https://github.com/sanger-pathogens/deployment/pull/125) ([andrewjpage](https://github.com/andrewjpage))
- extact\_modifications\_to\_plots [\#124](https://github.com/sanger-pathogens/deployment/pull/124) ([andrewjpage](https://github.com/andrewjpage))
- Adding transfer\_genes.pl and transfer\_ids\_to\_EMBL.pl to production bin [\#123](https://github.com/sanger-pathogens/deployment/pull/123) ([aslett1](https://github.com/aslett1))
- Added missing comma [\#122](https://github.com/sanger-pathogens/deployment/pull/122) ([carlacummins](https://github.com/carlacummins))
- Added snp\_sites\_plink\_files.py to repo [\#121](https://github.com/sanger-pathogens/deployment/pull/121) ([carlacummins](https://github.com/carlacummins))
- Removed final TraDIS scripts from general repo [\#120](https://github.com/sanger-pathogens/deployment/pull/120) ([carlacummins](https://github.com/carlacummins))
- Added find\_large\_files.py to mapping [\#119](https://github.com/sanger-pathogens/deployment/pull/119) ([carlacummins](https://github.com/carlacummins))
- Added run\_haploview script [\#118](https://github.com/sanger-pathogens/deployment/pull/118) ([aslett1](https://github.com/aslett1))
- signalp chado [\#117](https://github.com/sanger-pathogens/deployment/pull/117) ([andrewjpage](https://github.com/andrewjpage))
- pacbio\_smrtanalysis [\#116](https://github.com/sanger-pathogens/deployment/pull/116) ([andrewjpage](https://github.com/andrewjpage))
- Added NCTC tab generation script [\#115](https://github.com/sanger-pathogens/deployment/pull/115) ([aslett1](https://github.com/aslett1))
- Missing comma added [\#114](https://github.com/sanger-pathogens/deployment/pull/114) ([aslett1](https://github.com/aslett1))
- Removed iprscan\_chado from prod deployment and added new pair of interpr... [\#113](https://github.com/sanger-pathogens/deployment/pull/113) ([aslett1](https://github.com/aslett1))
- Added AUTHORS file [\#112](https://github.com/sanger-pathogens/deployment/pull/112) ([aslett1](https://github.com/aslett1))
- Added interpro\_to\_chado\_load to production bin [\#111](https://github.com/sanger-pathogens/deployment/pull/111) ([aslett1](https://github.com/aslett1))
- Added scout\_wrapper.pl to prod bin and auto\_update\_go\_terms.pl to pathde... [\#110](https://github.com/sanger-pathogens/deployment/pull/110) ([aslett1](https://github.com/aslett1))
- Adding reciprocal\_fasta.pl to production bin [\#109](https://github.com/sanger-pathogens/deployment/pull/109) ([aslett1](https://github.com/aslett1))
- Added cogblast.pl to prod bin [\#108](https://github.com/sanger-pathogens/deployment/pull/108) ([aslett1](https://github.com/aslett1))
- Added rm\_pathpipe\_tmp.py to mappings [\#107](https://github.com/sanger-pathogens/deployment/pull/107) ([carlacummins](https://github.com/carlacummins))
- Added run\_beast.pl to production deployment [\#106](https://github.com/sanger-pathogens/deployment/pull/106) ([aslett1](https://github.com/aslett1))
- Added rfamscan\_dechunk.pl [\#105](https://github.com/sanger-pathogens/deployment/pull/105) ([aslett1](https://github.com/aslett1))
- reconnect symlinks from nexsan to lustre [\#103](https://github.com/sanger-pathogens/deployment/pull/103) ([andrewjpage](https://github.com/andrewjpage))
- merge farm3 branch [\#102](https://github.com/sanger-pathogens/deployment/pull/102) ([andrewjpage](https://github.com/andrewjpage))
- Updated mappings. Removed tradis\_essentiality.R [\#101](https://github.com/sanger-pathogens/deployment/pull/101) ([carlacummins](https://github.com/carlacummins))
- Added the script trna-se.pl to production bin [\#100](https://github.com/sanger-pathogens/deployment/pull/100) ([aslett1](https://github.com/aslett1))
- Added prosite\_by\_perl.pl script to prod bin [\#99](https://github.com/sanger-pathogens/deployment/pull/99) ([aslett1](https://github.com/aslett1))
- Added find untracked studies script [\#98](https://github.com/sanger-pathogens/deployment/pull/98) ([aslett1](https://github.com/aslett1))
- Removed spurious class file [\#97](https://github.com/sanger-pathogens/deployment/pull/97) ([aslett1](https://github.com/aslett1))
- Added pfamscan2tab\_chado.pl script [\#96](https://github.com/sanger-pathogens/deployment/pull/96) ([aslett1](https://github.com/aslett1))
- Dollar sign in file name backslashed [\#95](https://github.com/sanger-pathogens/deployment/pull/95) ([aslett1](https://github.com/aslett1))
- Further class files added for java [\#94](https://github.com/sanger-pathogens/deployment/pull/94) ([aslett1](https://github.com/aslett1))
- Added google related class files to production java [\#93](https://github.com/sanger-pathogens/deployment/pull/93) ([aslett1](https://github.com/aslett1))
- Added crawl python modules [\#92](https://github.com/sanger-pathogens/deployment/pull/92) ([aslett1](https://github.com/aslett1))
- Added Crawl perl modules [\#91](https://github.com/sanger-pathogens/deployment/pull/91) ([aslett1](https://github.com/aslett1))
- Added orthomcl2tab.pl [\#90](https://github.com/sanger-pathogens/deployment/pull/90) ([aslett1](https://github.com/aslett1))
- Removed internal rfam\_scan script from path [\#89](https://github.com/sanger-pathogens/deployment/pull/89) ([aslett1](https://github.com/aslett1))
- Adding DaemonsLog [\#88](https://github.com/sanger-pathogens/deployment/pull/88) ([carlacummins](https://github.com/carlacummins))
- Added DaemonsLog to mapping [\#87](https://github.com/sanger-pathogens/deployment/pull/87) ([carlacummins](https://github.com/carlacummins))
- Added DaemonsLog to production mapping [\#86](https://github.com/sanger-pathogens/deployment/pull/86) ([carlacummins](https://github.com/carlacummins))
- Added dotplot script for David Harris [\#85](https://github.com/sanger-pathogens/deployment/pull/85) ([aslett1](https://github.com/aslett1))
- Removed remaining find scripts [\#84](https://github.com/sanger-pathogens/deployment/pull/84) ([carlacummins](https://github.com/carlacummins))
- Removed some find script mappings [\#83](https://github.com/sanger-pathogens/deployment/pull/83) ([carlacummins](https://github.com/carlacummins))
- Added get\_GO\_assosiations.pl, updated indentation [\#82](https://github.com/sanger-pathogens/deployment/pull/82) ([carlacummins](https://github.com/carlacummins))
- Added get\_GO\_assosiations.pl [\#81](https://github.com/sanger-pathogens/deployment/pull/81) ([carlacummins](https://github.com/carlacummins))
- Added get\_GO\_association.pl to mapping [\#80](https://github.com/sanger-pathogens/deployment/pull/80) ([carlacummins](https://github.com/carlacummins))
- Conflicts resolved [\#79](https://github.com/sanger-pathogens/deployment/pull/79) ([nds](https://github.com/nds))
- Scripts to prepare files for EMBL submission [\#78](https://github.com/sanger-pathogens/deployment/pull/78) ([nds](https://github.com/nds))
- annotate references [\#77](https://github.com/sanger-pathogens/deployment/pull/77) ([andrewjpage](https://github.com/andrewjpage))
- Update pacbio assembly script to use v2.0.1 or the new v2.1.0 [\#76](https://github.com/sanger-pathogens/deployment/pull/76) ([martinghunt](https://github.com/martinghunt))
- bacteria pan genome [\#75](https://github.com/sanger-pathogens/deployment/pull/75) ([andrewjpage](https://github.com/andrewjpage))
- Lib directory deployment [\#74](https://github.com/sanger-pathogens/deployment/pull/74) ([andrewjpage](https://github.com/andrewjpage))
- put adapters in /software [\#73](https://github.com/sanger-pathogens/deployment/pull/73) ([andrewjpage](https://github.com/andrewjpage))
- deploy adapters [\#72](https://github.com/sanger-pathogens/deployment/pull/72) ([andrewjpage](https://github.com/andrewjpage))
- Revert [\#71](https://github.com/sanger-pathogens/deployment/pull/71) ([carlacummins](https://github.com/carlacummins))
- Edit to place files in correct location [\#70](https://github.com/sanger-pathogens/deployment/pull/70) ([carlacummins](https://github.com/carlacummins))
- Update to place files in the correct place [\#69](https://github.com/sanger-pathogens/deployment/pull/69) ([carlacummins](https://github.com/carlacummins))
- Mapping fix test [\#68](https://github.com/sanger-pathogens/deployment/pull/68) ([carlacummins](https://github.com/carlacummins))
- identify clusters [\#67](https://github.com/sanger-pathogens/deployment/pull/67) ([andrewjpage](https://github.com/andrewjpage))
- identify protein changes at position [\#66](https://github.com/sanger-pathogens/deployment/pull/66) ([andrewjpage](https://github.com/andrewjpage))
- remove snp sites [\#65](https://github.com/sanger-pathogens/deployment/pull/65) ([andrewjpage](https://github.com/andrewjpage))
- plasmid identification [\#64](https://github.com/sanger-pathogens/deployment/pull/64) ([andrewjpage](https://github.com/andrewjpage))
- extract regions from multifasta [\#63](https://github.com/sanger-pathogens/deployment/pull/63) ([andrewjpage](https://github.com/andrewjpage))
- added new file chado\_dump\_genome [\#62](https://github.com/sanger-pathogens/deployment/pull/62) ([tcarver](https://github.com/tcarver))
- fasta manipulation scripts [\#61](https://github.com/sanger-pathogens/deployment/pull/61) ([andrewjpage](https://github.com/andrewjpage))
- Adding mapping for writedb\_entry\_set [\#60](https://github.com/sanger-pathogens/deployment/pull/60) ([carlacummins](https://github.com/carlacummins))
- Revert "added pacbio\_assemble\_smrtanalysis-2.0.1" [\#59](https://github.com/sanger-pathogens/deployment/pull/59) ([martinghunt](https://github.com/martinghunt))
- added pacbio\_assemble\_smrtanalysis-2.0.1 [\#58](https://github.com/sanger-pathogens/deployment/pull/58) ([martinghunt](https://github.com/martinghunt))
- added pacbio\_assemble\_smrtanalysis-2.0.1 [\#57](https://github.com/sanger-pathogens/deployment/pull/57) ([martinghunt](https://github.com/martinghunt))
- Added ropy python module to mapping [\#56](https://github.com/sanger-pathogens/deployment/pull/56) ([carlacummins](https://github.com/carlacummins))
- Updated mapping for python modules [\#55](https://github.com/sanger-pathogens/deployment/pull/55) ([carlacummins](https://github.com/carlacummins))
- Updated mapping for python modules [\#54](https://github.com/sanger-pathogens/deployment/pull/54) ([carlacummins](https://github.com/carlacummins))
- New script to strip sequence off gff files [\#53](https://github.com/sanger-pathogens/deployment/pull/53) ([nds](https://github.com/nds))
- rebalance disks [\#52](https://github.com/sanger-pathogens/deployment/pull/52) ([andrewjpage](https://github.com/andrewjpage))
- GPL [\#51](https://github.com/sanger-pathogens/deployment/pull/51) ([andrewjpage](https://github.com/andrewjpage))
- assembly stats to production [\#50](https://github.com/sanger-pathogens/deployment/pull/50) ([andrewjpage](https://github.com/andrewjpage))
- added find\_untracked\_studies.pl [\#49](https://github.com/sanger-pathogens/deployment/pull/49) ([CraigPorter](https://github.com/CraigPorter))
- move functionality for gene\_name\_occurances to Bio-AutomatedAnnotation [\#48](https://github.com/sanger-pathogens/deployment/pull/48) ([andrewjpage](https://github.com/andrewjpage))
- Added PathfindLog module [\#47](https://github.com/sanger-pathogens/deployment/pull/47) ([CraigPorter](https://github.com/CraigPorter))
- download blast dbs from refseq [\#46](https://github.com/sanger-pathogens/deployment/pull/46) ([andrewjpage](https://github.com/andrewjpage))
- gene name occurances [\#45](https://github.com/sanger-pathogens/deployment/pull/45) ([andrewjpage](https://github.com/andrewjpage))
- annotation find [\#44](https://github.com/sanger-pathogens/deployment/pull/44) ([andrewjpage](https://github.com/andrewjpage))
- infofind [\#43](https://github.com/sanger-pathogens/deployment/pull/43) ([CraigPorter](https://github.com/CraigPorter))
- fastq to fasta [\#42](https://github.com/sanger-pathogens/deployment/pull/42) ([andrewjpage](https://github.com/andrewjpage))
- merge rnaseq [\#41](https://github.com/sanger-pathogens/deployment/pull/41) ([andrewjpage](https://github.com/andrewjpage))
- added chado\_dump\_introns  [\#40](https://github.com/sanger-pathogens/deployment/pull/40) ([tcarver](https://github.com/tcarver))
- convert\_gff3\_to\_gff2.pl [\#39](https://github.com/sanger-pathogens/deployment/pull/39) ([andrewjpage](https://github.com/andrewjpage))
- assembly stats [\#38](https://github.com/sanger-pathogens/deployment/pull/38) ([andrewjpage](https://github.com/andrewjpage))
- Added list\_species\_sequenced.pl to pathdev bin. [\#37](https://github.com/sanger-pathogens/deployment/pull/37) ([CraigPorter](https://github.com/CraigPorter))
- tradis merge plots [\#36](https://github.com/sanger-pathogens/deployment/pull/36) ([andrewjpage](https://github.com/andrewjpage))
- validate multifasta files [\#35](https://github.com/sanger-pathogens/deployment/pull/35) ([andrewjpage](https://github.com/andrewjpage))
- deploy R scripts [\#34](https://github.com/sanger-pathogens/deployment/pull/34) ([andrewjpage](https://github.com/andrewjpage))
- tradis analysis script  [\#33](https://github.com/sanger-pathogens/deployment/pull/33) ([andrewjpage](https://github.com/andrewjpage))
- rename stats to avoid conflict [\#32](https://github.com/sanger-pathogens/deployment/pull/32) ([andrewjpage](https://github.com/andrewjpage))
- tradis [\#31](https://github.com/sanger-pathogens/deployment/pull/31) ([andrewjpage](https://github.com/andrewjpage))
- assemblyfind [\#30](https://github.com/sanger-pathogens/deployment/pull/30) ([andrewjpage](https://github.com/andrewjpage))
- bamcheck and plot bamcheck on list of bamfiles [\#29](https://github.com/sanger-pathogens/deployment/pull/29) ([andrewjpage](https://github.com/andrewjpage))
- species freq aw [\#28](https://github.com/sanger-pathogens/deployment/pull/28) ([andrewjpage](https://github.com/andrewjpage))
- Added Pathfind.pm module to production\_lib. [\#27](https://github.com/sanger-pathogens/deployment/pull/27) ([CraigPorter](https://github.com/CraigPorter))
- scaffold perl application [\#26](https://github.com/sanger-pathogens/deployment/pull/26) ([andrewjpage](https://github.com/andrewjpage))
- dont use git repository module for updating submodules [\#25](https://github.com/sanger-pathogens/deployment/pull/25) ([andrewjpage](https://github.com/andrewjpage))
- submodule support [\#24](https://github.com/sanger-pathogens/deployment/pull/24) ([andrewjpage](https://github.com/andrewjpage))
- splimer deployment [\#23](https://github.com/sanger-pathogens/deployment/pull/23) ([dg8](https://github.com/dg8))
- splimer dependancies [\#22](https://github.com/sanger-pathogens/deployment/pull/22) ([dg8](https://github.com/dg8))
- rna seq find [\#21](https://github.com/sanger-pathogens/deployment/pull/21) ([andrewjpage](https://github.com/andrewjpage))
- assemble lanes and retry failed lanes [\#20](https://github.com/sanger-pathogens/deployment/pull/20) ([andrewjpage](https://github.com/andrewjpage))
- remove temp directories when everything is finished [\#19](https://github.com/sanger-pathogens/deployment/pull/19) ([andrewjpage](https://github.com/andrewjpage))
- shuffle reads [\#18](https://github.com/sanger-pathogens/deployment/pull/18) ([andrewjpage](https://github.com/andrewjpage))
- snpfind [\#17](https://github.com/sanger-pathogens/deployment/pull/17) ([andrewjpage](https://github.com/andrewjpage))
- mh12 utils [\#16](https://github.com/sanger-pathogens/deployment/pull/16) ([andrewjpage](https://github.com/andrewjpage))
- assembly stats programs [\#15](https://github.com/sanger-pathogens/deployment/pull/15) ([andrewjpage](https://github.com/andrewjpage))
- Added map2bam.pl script. [\#14](https://github.com/sanger-pathogens/deployment/pull/14) ([CraigPorter](https://github.com/CraigPorter))
- reffind [\#13](https://github.com/sanger-pathogens/deployment/pull/13) ([andrewjpage](https://github.com/andrewjpage))
- forecast sequencing [\#12](https://github.com/sanger-pathogens/deployment/pull/12) ([andrewjpage](https://github.com/andrewjpage))
- deploy script to get stats on number of lanes at each stage in pipeline [\#11](https://github.com/sanger-pathogens/deployment/pull/11) ([andrewjpage](https://github.com/andrewjpage))
- mapfind [\#10](https://github.com/sanger-pathogens/deployment/pull/10) ([andrewjpage](https://github.com/andrewjpage))
- remove old file for looking up accessions [\#9](https://github.com/sanger-pathogens/deployment/pull/9) ([andrewjpage](https://github.com/andrewjpage))
- add accession find [\#8](https://github.com/sanger-pathogens/deployment/pull/8) ([andrewjpage](https://github.com/andrewjpage))
- Added generate\_ref\_index script to deployment [\#7](https://github.com/sanger-pathogens/deployment/pull/7) ([jacquikeane](https://github.com/jacquikeane))
- directory size report [\#6](https://github.com/sanger-pathogens/deployment/pull/6) ([andrewjpage](https://github.com/andrewjpage))
- New script to look up the accession number \(ERR...\) for each of the lanes specified in the input file \(lanes.list\) \[RT ticket \#224699\] [\#5](https://github.com/sanger-pathogens/deployment/pull/5) ([pajanne](https://github.com/pajanne))
- disk usage script [\#4](https://github.com/sanger-pathogens/deployment/pull/4) ([andrewjpage](https://github.com/andrewjpage))
- cleanup deployment [\#3](https://github.com/sanger-pathogens/deployment/pull/3) ([andrewjpage](https://github.com/andrewjpage))
- specific test setup for pathpipe [\#2](https://github.com/sanger-pathogens/deployment/pull/2) ([andrewjpage](https://github.com/andrewjpage))
- Near final deployment config settings [\#1](https://github.com/sanger-pathogens/deployment/pull/1) ([andrewjpage](https://github.com/andrewjpage))


